app/src/main/java/com/zulip/android/ZulipApp.java

```diff
@@ -261,4 +264,29 @@ public void markMessageAsRead(Message message) {
             unreadMessageHandler.sendEmptyMessageDelayed(0, 2000);
         }
     }
+    public void MuteTopic(int streamId, String subject) {
+        System.out.println((streamId + "-" + subject));
+        muted_topics.add(streamId + "-" + subject);
+        SharedPreferences.Editor editor = settings.edit();
+        editor.putStringSet("muted_topics", muted_topics);
+        editor.commit();        
+    }
+
+    public void UnMuteTopic(int streamId, String subject) {
+        muted_topics.remove(streamId + "-" + subject);
+        SharedPreferences.Editor editor = settings.edit();
+        editor.putStringSet("muted_topics", muted_topics);
+        editor.commit();
+    }
+
+    public boolean isTopicMute(int streamId, String subject) {
+        if (muted_topics.contains(streamId + "-" + subject))
```

Comment:
the `if` here is superfluous. you can simply return ie:

    return muted_topics.contains(streamId + "-" + subject)